crates/federate/src/worker.rs

```diff
@@ -232,10 +232,7 @@ impl InstanceWorker {
     } else {
       // this is the initial creation (instance first seen) of the federation queue for this
       // instance
-
-      // skip all past activities:
-      self.state.last_successful_id = Some(latest_id);
-      // save here to ensure it's not read as 0 again later if no activities have happened
+      self.state.last_successful_id = Some(ActivityId(0));
```

Comment:
This line was causing the api test to fail, because the [follow from epsilon](https://github.com/LemmyNet/lemmy/pull/5367/files#diff-13962f3f5cbd731da009a3219d9b793236ad9f98b1f9eed2c4f49d2a8684ec32R690) was not being sent. So it should really go through all activities here and check if there is one for the target instance.

cc @phiresky

Reply:
Added a workaround for this now, by fetching a user first in order to initialize the federation queue.